packages/commerce/package.json

```diff
@@ -8,12 +8,14 @@
   "types": "./dist/index.d.ts",
   "source": "./src/index.ts",
   "files": [
-    "dist/**"
+    "dist/**",
+    "dist/css/commerce.css"
   ],
   "scripts": {
     "test": "jest",
     "dev": "etsc",
-    "build": "tsc --jsx react-jsx --outDir dist",
+    "build": "tsc --jsx react-jsx --outDir dist && pnpm build:css",
+    "build:css": "NODE_ENV=production postcss ./src/styles/commerce.css -o ./dist/css/commerce.css",
```

Comment:
straight forward build step here to get the css

Reply:
why do you want distribute all of tailwind here? Isn't commerce only going to be effectively used by next.js projects (i.e. apps/*)? There are no custom styles so this is unecessary I think

Reply:
it's purged so it only includes what is used, but I have no idea what I'm doing and there are no examples so I'm just guessing 

the bit where I just point purge at these source trees is probably better for our use case, but won't work if we distribute these libraries